identity/Cargo.toml

```diff
@@ -51,7 +51,11 @@ send-sync-storage = ["identity-account-storage/send-sync-storage"]
 storage-test-suite = ["identity-account-storage/storage-test-suite"]
 
 # Enables revocation with `RevocationBitmap2022`.
-revocation-bitmap = ["identity-account?/revocation-bitmap", "identity-iota/revocation-bitmap"]
+revocation-bitmap = [
+  "identity-account?/revocation-bitmap",
```

Comment:
This optional feature syntax is only available in Rust 1.60+, need to bump the MSRV.

Reply:
Should also add `revocation-bitmap` to the `default` feature sets.

Reply:
This seems like a nice application of the new syntax, but I'm also fine not using it.

Reply:
Changed MSRV to 1.60.